Composer/packages/client/src/constants.tsx

```diff
@@ -358,14 +367,11 @@ export const selectIntentDialog = {
 
 export const enableOrchestratorDialog = {
   get title() {
-    return formatMessage('Enable Orchestrator Recognizer');
+    return formatMessage('Enable Orchestrator');
   },
   get subText() {
-    return formatMessage('Enable Orchestrator as the recognizer for routing to other skills');
-  },
-  get content() {
     return formatMessage(
-      'Multi-bot projects work best with the Orchestrator recognizer set at the dispatching dialog (typically the root dialog). Orchestrator helps identify and dispatch user intents from the root dialog to the respective skill that handles the intent. Orchestrator does not support entity extraction. If you plan to combine entity extraction and routing at the root dialog, use LUIS instead.'
+      'A bot that consists of multiple bots or connects to skills (multi-bot project) needs Orchestrator to detect and route user input to the appropriate bot or skill.'
```

Comment:
@vivekkshankar , when we did the second pass on this text, we were trying to convey that we **recommend** Orchestrator for this scenario (after all, the user can say no to using it), but the third pass here has changed back to **need** orchestrator... Is this right?

Reply:
FYI - This is on line 136 on the "Add a skill" tab of the shared spreadsheet. I'm taking all of my changes from there.

Reply:
Yeah, sorry I should've put that comment in that spreadsheet. I'll do that instead.